### DISCUSSION TITLE
Sharing to Expo app

### DISCUSSION THREAD

**Discussion by brunolcarlos** [expo/expo#16613] [Proposals and Ideas]
Hi guys, some news about this requested feature?

https://expo.canny.io/feature-requests/p/share-extension-ios-share-intent-android

**Comment by barthap** [comment]
AFAIK, it's planned, but no ETA

**Reply by shamimalpha** [reply]
As things stand, is it possible to pull it off using just expo plugins and mods or we must use expo prebuild?

**Reply by joseito-terence** [reply]
Any update on this?

**Reply by barthap** [reply]
Nothing official yet, but it's worth looking at this thread: https://github.com/meedan/react-native-share-menu/issues/162

**Comment by Nantris** [comment]
Any update here? This is Expo's single largest weakness, and has been for many years.

I'm guessing it must not be easy to implement, but I'd welcome more communication from the Expo team.

**Reply by achorein** [reply]
👋  got a working android and ios auto-config demo here https://github.com/achorein/expo-share-intent-demo, while waiting for the official expo package.

**Reply by Nantris** [reply]
Wow this looks really cool! Does it work with EAS?

**Reply by achorein** [reply]
Yes, i'm using it for a classic eas build :)

**Comment by MaxAst** [comment]
Hey all, I've developed a solution for iOS, which is pretty easy to set up and contains a couple of examples in the repo: https://github.com/MaxAst/expo-share-extension

**Reply by Karthik-B-06** [reply]
Hey @MaxAst I have couple of doubts, I am not able to access the AsyncStorage from the Share Intent component, trying to just upload whatever is being shared to my backend, without taking the user to the app. Am I missing something here? 

PS: Great package, I hope it is added to the expo documentation as well.
